data/sql/updates/pending_db_world/rev_1713004478213792900.sql

```diff
@@ -0,0 +1,292 @@
+
+DELETE FROM `creature` WHERE (`id1` = 4299) AND (`guid` IN (40008));
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(40008, 4299, 0, 0, 189, 0, 0, 1, 1, 1, 1081.6327, 1379.4738, 30.445854, 3.1230602264404297, 86400, 0, 1, 2901, 2014, 2, 0, 0, 0, '', 0);
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (40008));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(40008, 0, 0, 0, 0, 0, 0, NULL);
+
+SET @PATH := 4000800;
+DELETE FROM `waypoints` WHERE `entry`= @PATH;
+INSERT INTO `waypoints` (`entry`, `pointid`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `point_comment`) VALUES
+(@PATH, 0+1, 1081.633, 1379.474, 30.44585, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 1+1, 1081.354, 1385.916, 30.44585, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 2+1, 1081.633, 1379.474, 30.44585, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 3+1, 1100.825, 1379.118, 30.44586, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 4+1, 1128.136, 1379.364, 30.47259, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 5+1, 1137.428, 1373.87, 30.43785, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 6+1, 1140.826, 1345.011, 30.44586, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 7+1, 1137.428, 1373.87, 30.43785, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 8+1, 1128.276, 1379.281, 30.44585, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 9+1, 1100.825, 1379.118, 30.44586, NULL, 0,'Scarlet Monastery Patrol'),
+(@PATH, 10+1, 1081.633, 1379.474, 30.44585, NULL, 0,'Scarlet Monastery Patrol');
+
+UPDATE `creature` SET  `MovementType`=0 WHERE `guid`=40008;
+
+
+
+ -- Scarlet Wizard with guid 40034 smart ai
+SET @ENTRY := -40034;
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = 4300;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryOrGuid` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 0, 0, 0, 1, 0, 100, 0, 0, 0, 0, 0, 53, 0, 4003400, 1, 0, 0, 2, 1, 0, 0, 0, 0, 0, 0, 0, 'Once (OOC) - Self: Start path #4003400, walk, repeat, Aggressive');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = -40034 AND `SourceId` = 0;
```

Comment:
What does this refer to?

Reply:
I don't know why this thing exists. This is auto-generated. I can delete it if I don't need it